jiant/tasks/lm.py

```diff
@@ -172,3 +178,102 @@ def __init__(self, path, *args, **kw):
             "val": os.path.join(path, "valid.sentences.txt"),
             "test": os.path.join(path, "test.sentences.txt"),
         }
+
+
+@register_task("mlm", rel_path="WikiText103/")
```

Comment:
Requesting rename: `mlm` task name seems overly broad (there can be other `mlm`-style tasks, right?). Borrow the naming pattern from [another task that uses WikiText103 data](https://github.com/nyu-mll/jiant/blob/61440732f4df2c54e68e59ead34b8656bf52af3b/jiant/tasks/tasks.py#L2129) where the name mentions the data and model-type?